src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Do List </title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+  <main class="d-flex">
+    <section class="d-flex cols todo-list">
+        
+      <header class="header d-flex s-between">
+          <h2>Today's To Do </h2>
+          <!-- <span class="material-icons" id="delete-all">autorenew</span> -->
+      </header>
+
+      <form action="#" class="d-flex s-between" id="add-task">
+          <input type="text" name="activity" placeholder="Add your List">
+          <!-- <button type="submit">
+              <span class="material-icons">keyboard_return</span>
+          </button> -->
```

Comment:
These lines of code should also be removed to make your code looks professional. Kindly remove it. 👍🏾